styles.css

```
@@ -238,4 +235,103 @@ select {
     font-size: 12px;
     line-height: 140%;
     padding: 64px 98px 32px 98px;
+    max-width: 374px;
+}
+
+@media (min-width:855px) {
+    .about_blog {
+        max-width: 854px;
+        font-size: 54px
+    }
+
+    .articles {
+        font-size: 44px;
+        padding-bottom: 39px;
+    }
+
+    .small_text_about_blog {
+        font-size: 20px;
+        max-width: 550px;
+    }
+
+    .photo {
+        max-width: 640px;
+        flex-wrap: wrap;
+        height: 920px;
+        gap: 32px;
+        justify-content: space-between;
+    }
+
+    .attribute {
+        flex: 1;
+        max-width: 304px;
+        position: relative;
+        height: 338px;
+    }
+
+    .second_page_photos {
+        margin: 0px 0px 0px 0px;
+    }
+
+    .text_after_images {
+        padding: 20px 0px 0px 0px;
+        margin-left: auto;
+        margin-right: auto;
+        width: 282px;
+    }
+
+    .square_first_text {
+        left: 50%;
+        transform: translateX(-50.0%);
+    }
+
+    .scroll_text {
+        font-size: 22px;
+        letter-spacing: 0.1em;
+    }
+
+    .lorem {
+        font-size: 16px;
+        max-width: 482px;
+    }
+
+    .link {
+        font-size: 16px;
+    }
+}
+
+@media (min-width:1024px) {
+    .desktop_menu {
+        display: flex;
+        gap: 32px;
+        font-family: 'New York Small';
+        font-weight: 500;
+        font-size: 20px;
+        line-height: 170%;
+        letter-spacing: 0.12em;
+        text-transform: uppercase;
+        padding: 59px 61px 0px 0px;
+    }
+
+    .first_part {
+        border-bottom: 2px solid #000000;
+        padding-bottom: 40px;
+        color: #000000;
+        text-decoration: none
+    }
+
+    .second_part,
+    .third_part,
+    .fouth_part {
+        color: #000000;
+        text-decoration: none
+    }
```

1. `second_part`, `third_part`, `fourth_part` all are the same. you should use single class for them
2. in fact `first_part` also has the same styles, the only difference should be `border` and it is applied on active menu item. `padding` should be also applied to every link
```suggestion
    .menu-link {
        padding-bottom: 40px;
        color: #000000;
        text-decoration: none
    }
    
    .menu-link.active {
        border-bottom: 2px solid #000000;
    }
```